src/index.js

```diff
@@ -46,47 +44,58 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z]+_[A-Z][A-Z]+$'],
     indent: ['error', 2, {
       SwitchCase: 1
     }],
-    'key-spacing': 'off',
-    'keyword-spacing': ['off', {
+    'key-spacing': 'error',
+    'keyword-spacing': ['error', {
       overrides: {
         case: {
           after: true
         },
+        catch: {
+          after: true,
```

Comment:
The default values for `after` and `before` are both `true`, so I guess all these lines can be deleted.